pyanaconda/iutil.py

```diff
@@ -990,7 +991,7 @@ def strip_accents(s):
 
     """
     return ''.join((c for c in unicodedata.normalize('NFD', s)
-                      if unicodedata.category(c) != 'Mn'))
+                   if unicodedata.category(c) != 'Mn'))
```

Comment:
Fixed - it's interesting how the code sometimes almost organically mutates to accumulate stuff like this. :)